common/client-core/src/client/real_messages_control/real_traffic_stream.rs

```diff
@@ -259,6 +265,13 @@ where
 
         if let Err(err) = self.mix_tx.send(vec![next_message]).await {
             log::error!("Failed to send: {err}");
+        } else {
+            let event = if fragment_id.is_some() {
+                PacketStatisticsEvent::RealPacketSent
```

Comment:
also, for real packets, i think it might be cool to know what packet sized they used

Reply:
Agree!